lib/elixir/lib/file.ex

```diff
@@ -1388,4 +1393,18 @@ defmodule File do
     do: path
   defp maybe_to_string(path),
     do: IO.chardata_to_string(path)
+
+  @doc false
+  def bom_length(<<239, 187, 191, _rest::binary>>),
```

Comment:
If we are not using this function here, we can move it to the File.Stream module.